src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if UNIX
```

Comment:
It would be better to update the existing `Computer.cs` and `#if UNIX` sections that apply to non-Windows (removing the global `#if !UNIX` at the top).  This way `Restart-Computer` is also just a call to the `shutdown` command.

Reply:
I tried this. >90% of Computer.cs is Windows specific. So the code becomes really, really ugly. 

It's your call, ultimately. It's do-able, but ugly imho. Please let me know if you still want me to do it.

Reply:
I could also splint into:

* `Computer.cs`
* `Computer.Windows.cs`
* `Computer.Unix.cs`

Reply:
If the majority of the code is Windows specific it would make it harder to maintain trying to combine them, so I agree to keep it separate.  Thanks for looking into it.